### PR TITLE
Monkey Energy Overdose

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Gives monkey energy an overdose that makes you spout random phrases that make you sound like a boomer.

## Why It's Good For The Game
Makes it quite obvious when you have been chugging unhealthy amounts of monkey energy.

## Changelog
:cl:
add: overdose effect to monkey energy
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
use the strings framework you liberal
https://github.com/tgstation/tgstation/blob/master/code/__HELPERS/_string_lists.dm

**Comment by Paxilmaniac** [comment]
it uses the strings framework now

File: strings/boomer.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "boomer": [
+	"@pick(kids) these days have it too easy!",
+	"Do I look like I know what a @pick(file) is!?",
+	"How do I open @pick(file) again?",
+	"Unlike you snowflakes, I'm not offended as easily.",
```
Comment (cacogen) [inline strings/boomer.json]:
```suggestion
	"Unlike you snowflakes, I'm not offended so easily.",
```

File: strings/boomer.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+    "boomer": [
+	"@pick(kids) these days have it too easy!",
+	"Do I look like I know what a @pick(file) is!?",
+	"How do I open @pick(file) again?",
```
Comment (cacogen) [inline strings/boomer.json]:
```suggestion
	"How do I open a @pick(file) again?",
```

File: strings/boomer.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    ],
+
+    "kids": [
+	"kids",
+	"millenials",
+	"snowflakes"
+    ],
+
+    "file": [
+	"jpeg",
```
Comment (cacogen) [inline strings/boomer.json]:
```suggestion
	"JPEG",
	"PDF",
	"DMI",
	"JSON"
```

File: strings/boomer.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
+{
+    "boomer": [
+	"@pick(kids) these days have it too easy!",
+	"Do I look like I know what a @pick(file) is!?",
+	"How do I open @pick(file) again?",
+	"Unlike you snowflakes, I'm not offended as easily.",
+	"Back in my day..",
```
Comment (cacogen) [inline strings/boomer.json]:
```suggestion
	"Back in my day...",
	"When I was your age...",
```

File: strings/boomer.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+	"When I was your age..",
+	"This generation can't take a joke.",
+	"Why are @pick(kids) these days so @pick(sad) all the time?",
+	"These damn @pick(kids) need to get a @pick(impossible) for once!"
```
Comment (cacogen) [inline strings/boomer.json]:
```suggestion
	"These damn @pick(kids) need to get a @pick(impossible) for once!",
	"It's simply a matter of showing up, looking the manager in the eye, giving him a firm handshake and telling him you want the job."
```

**Comment by Paxilmaniac** [comment]
very cool cacogen

**Comment by cacogen** [comment]
im sry

**Comment by Paxilmaniac** [comment]
more boomer phrases good

File: strings/boomer.json [hunk 6]
```diff
@@ -0,0 +1,45 @@
+{
+    "boomer": [
+	"@pick(kids) these days have it too easy!",
+	"Do I look like I know what a @pick(file) is!?",
+	"How do I open a @pick(file) again?",
+	"Unlike you snowflakes, I'm not offended so easily.",
+	"Back in my day...",
+	"When I was your age...",
+	"When I was your age..",
```
Comment (cacogen) [inline strings/boomer.json]:
```suggestion
```

File: strings/boomer.json [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+	"millenials",
+	"snowflakes"
+    ],
+
+    "file": [
+	"JPEG",
+	"PDF",
+	"DMI",
+	"JSON"
+	"pdf",
```
Comment (cacogen) [inline strings/boomer.json]:
```suggestion
```

File: strings/boomer.json [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+	"dmi",
```
Comment (cacogen) [inline strings/boomer.json]:
```suggestion
```

File: strings/boomer.json [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+	"json"
```
Comment (cacogen) [inline strings/boomer.json]:
```suggestion
```

**Review by Cyberboss — DISMISSED** [review]

**Comment by HomocidlePancakes** [comment]
Ok, boomer.

File: strings/boomer.json [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+	"Do I look like I know what a @pick(file) is!?",
+	"How do I open a @pick(file) again?",
+	"Unlike you snowflakes, I'm not offended so easily.",
+	"Back in my day...",
+	"When I was your age...",
+	"This generation can't take a joke.",
+	"Why are @pick(kids) these days so @pick(sad) all the time?",
+	"These damn @pick(kids) need to get a @pick(impossible) for once!",
+	"It's simply a matter of showing up, looking the manager in the eye, giving him a firm handshake and telling him you want the job."
+    ],
```
Comment (TheVekter) [inline strings/boomer.json]: "You'll never get anywhere in life without a degree."
"Healthcare isn't really _that_ expensive."

**Comment by Ryll-Ryll** [comment]
Saying millennials might be a bit too on the nose considering they're halfway into the millennium

**Comment by nicbn** [comment]
You could use some word like midmillenials (cuz what Ryll Ryll said)

File: strings/boomer.json (hunk 10 above)
Comment (Paxilmaniac) [reply strings/boomer.json]: yes

File: strings/boomer.json [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+	"Unlike you snowflakes, I'm not offended so easily.",
+	"Back in my day...",
+	"When I was your age...",
+	"This generation can't take a joke.",
+	"Why are @pick(kids) these days so @pick(sad) all the time?",
+	"These damn @pick(kids) need to get a @pick(impossible) for once!",
+	"It's simply a matter of showing up, looking the manager in the eye, giving him a firm handshake and telling him you want the job.",
+	"You'll never get anywhere in life without a degree.",
+	"@pick(expense) isn't really that expensive.",
+	"When I was a kid I had to walk to school uphill both ways!"
```
Comment (Iamgoofball) [inline strings/boomer.json]: *in the snow

File: strings/boomer.json [hunk 12]
```diff
@@ -0,0 +1,52 @@
+{
+    "boomer": [
```
Comment (Iamgoofball) [inline strings/boomer.json]: "You need to pull yourself up by the bootstraps and (get a good job, start investing your money, make your credit score great early)."

Comment (TheVekter) [reply strings/boomer.json]: Add "build your 401k for retirement". I hear that one a lot.

Comment (Iamgoofball) [reply strings/boomer.json]: yes, 401k, how could I forget

Comment (Iamgoofball) [reply strings/boomer.json]: need something about "buying everything on a credit card so you're used to paying off your debts on time" as well, classic boomer advice right there

**Pull request merged**